templates/blog/comment_create.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h1>Додати коментар до поста</h1>
+
+  <form method="post" action="{% url 'blog:create' post.id %}">
```

Comment:
The URL pattern name 'blog:create' might be incorrect if the intention is to create a comment. Ensure that the URL pattern name corresponds to the correct view for creating a comment.